SW-Configuration.md

```diff
@@ -180,6 +183,14 @@ But when you uncomment the following line, then the AC is switched on, once you
 //#define POWERON_WHEN_CHANGING_MODE true           // uncomment it to switch on the AC when the mode (heat, cool, dry etc.) is changed
 ```
 
+## Using extended frame size for enabling 3D auto and vanes L/R ([support.h](src/support.h))
+Per default the 3D auto and vanes left/right is not supported. 
+But when you uncomment the following line, the frame size is extended to 33 bytes (like the WR-RAC module). This will make it possible to use 3D auto and vanes L/R.
```

Comment:
```suggestion
But when you uncomment the following line, the frame size is extended to 33 bytes (like the WF-RAC module). This will make it possible to use 3D auto and vanes L/R.
```